### DISCUSSION TITLE
Add custom github domain

### DISCUSSION THREAD

**Discussion by audiotonewastaken** [community/community#23530] [New to GitHub]
<p>I’m trying to add the domain Avack.xyz to <a href="https://github.com/audiotonewastaken/AvackGames" rel="noopener nofollow ugc">this</a> Github repo, but I’m having trouble with creating an A record on the Bluehost DNS editor. I put the Host record as @ and made it point to 185.199.108.153 with a TTL of 4 hours. will this work or is there something wrong with how I have set it up?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s <code>avackgames.xyz</code> not <code>Avack.xyz</code></p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077578-375a3f32-d391-4b75-85fe-d628d58ba38c.png" alt="image" data-base62-sha1="cc9rDMLihZVN4jNUom169cPXUpo" width="419" height="324"></p>
<p>at your CNAME, it’s still <code>avackgames.xyz</code></p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077580-746ae1b0-26dd-42b7-b15b-0b615d812826.png" alt="image" data-base62-sha1="4KcYnUSFvp7jwY8QLjLOD46JJMo" width="406" height="401"></p>
